resources/main.mjs

```diff
@@ -6,6 +6,7 @@ import { params } from "./params.mjs";
 
 // FIXME(camillobruni): Add base class
 class MainBenchmarkClient {
+    dev = false;
```

Comment:
"dev" is a really generic name. Can we instead call it `isInDeveloperMode`?

Reply:
Maybe we meet in the middle - `devMode` or `developerMode`? More descriptive than `dev` but also still <= 2 words like other params.

Reply:
Renamed to developerMode in the latest push - lmk if that works for you